leptos/src/error_boundary.rs

```diff
@@ -64,7 +64,25 @@ where
     provide_context(cx, errors);
 
     // Run children so that they render and execute resources
-    let children = children(cx).into_view(cx);
+    let children = children(cx);
+
+    #[cfg(all(debug_assertions, feature = "hydrate"))]
+    {
+        use leptos_dom::View;
+        if children.nodes.iter().any(|child| {
+            matches!(child, View::Suspense(_, _))
+            || matches!(child, View::Component(repr) if repr.name() == "Transition")
```

Comment:
would  this produce a false positive when the user names a custom component `Transition`?

Reply:
Yes. At that point, I honestly don't know what to say.

Reply:
could we use downcast_ref like already done in other places within leptos?

Reply:
I agree